spec/rollout_spec.rb

```diff
@@ -108,33 +108,36 @@
   ##
   shared_examples "User's Feature Checker" do
     context 'when feature is rolled out for the user' do
-      subject { rollout(feature_name, user) }
-      it { is_expected.to be_active(feature_name, user) }
+      subject { rollout(feature, user) }
```

Comment:
Is this value truly a feature or only the description of a feature (name or ID or something like that). If it is a string or a symbol, then I infer that it is not an entire feature, but just a name for it. In that case, I would keep the name `feature_name`.

Reply:
I'm agree!